public/app/features/dashboard/state/DashboardModel.ts

```diff
@@ -1135,6 +1135,10 @@ export class DashboardModel implements TimeModel {
     return this.originalTime;
   }
 
+  isPublicDashboard(): boolean {
+    return this.meta.publicDashboardAccessToken !== undefined && this.meta.publicDashboardAccessToken !== '';
```

Comment:
Could we do `this.meta.publicDashboardAccessToken ?? ""` just in case it ends up being a different falsy value?

Reply:
changed to use `getConfig().isPublicDashboardView` instead